File: Modul-3/Soal-1/PRAK301_2410817310012_FAISALTANJUNG.c
#include <stdio.h>
int main(){
    int bilangan1, bilangan2, bilangan3;

    printf("");
    scanf("%d %d %d", &bilangan1 , &bilangan2, &bilangan3);

    if(bilangan1 > bilangan2 & bilangan2 > bilangan3){ // 1 > 2 > 3
        printf("%d %d %d\n", bilangan3, bilangan2, bilangan1);
    }
    else if(bilangan1 > bilangan3 & bilangan3 > bilangan2){ // 1 > 3 > 2
        printf("%d %d %d\n", bilangan2, bilangan3, bilangan1);
    }
    else if(bilangan2 > bilangan1 & bilangan1 > bilangan3){ // 2 > 1 > 3
        printf("%d %d %d\n", bilangan3, bilangan1, bilangan2);
    }
    else if(bilangan2 > bilangan3 & bilangan3 > bilangan1){ // 2 > 3 > 1
        printf("%d %d %d\n", bilangan1, bilangan3, bilangan2);
    }
    else if(bilangan3 > bilangan1 & bilangan1 > bilangan2){ // 3 > 1 > 2
        printf("%d %d %d", bilangan2, bilangan1, bilangan3);
    }
    else{ // 3 > 2 > 1
        printf("%d %d %d\n", bilangan1, bilangan2 , bilangan3);
    }
}

File: Modul-3/Soal-1/PRAK301_2410817310012_FAISALTANJUNG.py
bilangan1,bilangan2,bilangan3 =input().split()


if bilangan1 > bilangan2 and bilangan2 > bilangan3: # 1 > 2 > 3
    print(f"{bilangan3} {bilangan2} {bilangan1}")
elif bilangan1 > bilangan3 and bilangan3 > bilangan2 : # 1 > 3 > 2
    print(f"{bilangan2} {bilangan3} {bilangan1}")
elif bilangan2 > bilangan1 and bilangan1 > bilangan3 : # 2 > 1 > 3
    print(f"{bilangan3} {bilangan1} {bilangan2}")
elif bilangan2 > bilangan3 and bilangan3 > bilangan1 : # 2 > 3 > 1
    print(f"{bilangan1} {bilangan3} {bilangan2}")
elif bilangan3 > bilangan1 and bilangan1 > bilangan2 : # 3 > 1 > 2
    print(f"{bilangan2} {bilangan1} {bilangan3}")
elif bilangan3 > bilangan2 and bilangan2 > bilangan1: # 3 > 2 > 1
    print(f"{bilangan1} {bilangan2} {bilangan3}")